README.md

```diff
@@ -1,18 +1,46 @@
 # k
 A general command line client for Apache Kafka.
 
-## Build
+## Install
 
 ```
+# Set a directory where Go packages will be installed.
+export GOPATH=$HOME/gocode
+
+# Install `k` and dependencies.
+go get github.com/xstevens/k
+```
+
+This will make `k` available as `$GOPATH/bin/k`.
+Add `$GOPATH/bin` to your `$PATH` to have it available as `k`.
```

Comment:
The above is an attempt to make this more accessible to people like me who are not very familiar with Go and its tooling. This seemed to be the simplest path to go from zero to a working `k` build.